monkey/infection_monkey/post_breach/pba.py

```diff
@@ -35,7 +35,7 @@ def run(self):
                 T1064Telem(
                     ScanStatus.USED, f"Scripts were used to execute {self.name} post breach action."
                 ).send()
-            return PostBreachData(self.name, self.command, result)
+            yield PostBreachData(self.name, self.command, result)
```

Comment:
I'm not sure I understand this change. The function does nothing after yielding, so it is, in effect, the same as a return.